pyaedt/modeler/Circuit.py

```diff
@@ -131,21 +147,24 @@ def edb(self):
 
     @property
     def layouteditor(self):
+        """Layout editor.""""
         if self._parent.design_type == "Twin Builder":
             return
         return self.odesign.SetActiveEditor("Layout")
 
     @property
     def model_units(self):
+        """Model units."""
         return retry_ntimes(10, self.layouteditor.GetActiveUnits)
 
     @property
     def primitives(self):
-        """
+        """Primitives.
 
         Returns
         -------
         Primitives3DLayout
```

Comment:
```suggestion
        :class:`pyaedt.modeler.Primitives3DLayout`
```
I ended up having to search for this one.  You can do this from the github search by searching for the class definition (search term is "class Primitives3DLayout") and then finding out where it's defined:
![tmp](https://user-images.githubusercontent.com/11981631/125670380-7d225ddf-f221-4914-9de0-7cbb30e7e940.png)

Note that instead of `/`, python uses `.` to denote submodules.  Hence why it's `pyaedt.modeler.Primitives3DLayout`

Reply:
@akaszynski  Thanks for this great explanation. I'm fairly certain I get it now!